CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2024-05-30 - v6.8.0
```

Comment:
We should remember to update during the release

Reply:
Good call.